lib/index.js

```diff
@@ -0,0 +1,127 @@
+// MiniLightbox
```

Comment:
Can you remove, please, these three lines (1 to 3)? These were added in the old days. Actually this project is one of my first open-source projects I created. :smile: